Graphics/ShaderTools/src/ShaderToolsCommon.cpp

```diff
@@ -173,47 +175,48 @@ static void ExtractDependencies(std::function<void(const Diligent::String&)> Inc
         None,
         AfterHash,
         AfterInclude,
-        InsideIncludeBrackets
+        InsidePointyIncludeBrackets,
+        InsideStraightIncludeBrackets
     } PreprocessorState = None;
     size_t Offset       = 0;
 
-    // Find positions of the first hash and slash
-    const char* NextHash  = static_cast<const char*>(memchr(pBuffer + Offset, '#', BufferSize - Offset));
-    const char* NextSlash = static_cast<const char*>(memchr(pBuffer + Offset, '/', BufferSize - Offset));
+    /// Find positions of the first hash and slash
+    const Char* NextHash  = static_cast<const char*>(memchr(pBuffer + Offset, '#', BufferSize - Offset));
+    const Char* NextSlash = static_cast<const char*>(memchr(pBuffer + Offset, '/', BufferSize - Offset));
     size_t      Start     = 0;
 
-    //We iterate over the characters of the buffer
+    /// We iterate over the characters of the buffer
     while (Offset < BufferSize)
     {
         switch (PreprocessorState)
         {
             case None:
             {
-                // Find a hash character if the current position is greater NextHash
+                /// Find a hash character if the current position is greater NextHash
```

Comment:
`///` Is for doxygen. These comment are implementation details. They don't need to be in the documentation.